### ISSUE TITLE
Exception calling "Save" with "0" argument(s): "Error saving file

### ISSUE THREAD

**Issue by namnamir** [dfinke/ImportExcel#687]
I have faced a strange issue. I have a function to create woorksheet for each dataset that I have.
It sometimes work fine and sometime not. The error is:

```powershell
Exception calling "Save" with "0" argument(s): "Error saving file
C:\Users\xxxxxxx\YYYYYYY_02-10-2019-01-10-22.xlsx"
At line:90 char:9
+         $TEMP.save()
+         ~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : InvalidOperationException
```

I have found that the error only occurs when the number of columns/rows are high (in my case 28 rows and 52 columns).
I also have found if I comment `Add-ExcelTable` line of the code, it works even for higher number of rows/columns (in my case 197 rows and 205 columns).

I do not know if it is a bug or I have made a mistake.

Here is my code:

```powershell
$TEMP = $null | Export-Excel -Path $OutputPath -KillExcel -WorkSheetname $Worksheet -PassThru
$WS = $TEMP.Workbook.Worksheets[$Worksheet]

[...OTHER STUFF...]

$Last_Row = $WS.Dimension.Rows
$Last_Column = (Get-ExcelColumnName $WS.Dimension.Columns).ColumnNumber
    
Add-ExcelTable -Range $WS.cells[1, 1, $Last_Row, $Last_Column] -TableName $Worksheet -TableStyle Medium5 -ShowHeader:$true -ShowFilter:$true -ShowRowStripes:$true

$TEMP.save()
```

**Comment by dfinke** [comment]
What happens if you use `Close-ExcelPackage $TEMP` instead of `$TEMP.save()`

**Comment by namnamir** [comment]
> What happens if you use `Close-ExcelPackage $TEMP` instead of `$TEMP.save()`

@dfinke I have checked it, it is the same.

**Comment by dfinke** [comment]
Could you possibly post the data you are working with?

**Comment by namnamir** [comment]
@dfinke It is a customized format of users in AD (Get-ADUsers) which is merged with some other columns like (specification of the computer she has used to login, e.g. Get-ADComputers)

**Comment by Damup1988** [comment]
Hey, I have the same issue.
I can say that it doesn't depend on your table size.
For example in the case below it also throws the error:

Export-Excel -Path "C:\Temp\_test\new.xlsx"
$x = Open-ExcelPackage -Path "C:\Temp\_test\new.xlsx"
$g = $x.Workbook.Worksheets.Item(1)
Add-ExcelTable -Range $g.cells["A1:B2"] -TableName "mytable_1" -TableStyle Medium11
$x.Save() #Error

**Comment by Damup1988** [comment]
> Hey, I have the same issue.
> I can say that it doesn't depend on your table size.
> For example in the case below it also throws the error:
> 
> Export-Excel -Path "C:\Temp_test\new.xlsx"
> $x = Open-ExcelPackage -Path "C:\Temp_test\new.xlsx"
> $g = $x.Workbook.Worksheets.Item(1)
> Add-ExcelTable -Range $g.cells["A1:B2"] -TableName "mytable_1" -TableStyle Medium11
> $x.Save() #Error

I've just tested using version 6.1.0 and it works fine

**Comment by namnamir** [comment]
I still have this issue.

I've solved the issue by changing the name of worksheets :) It is funny but it works only if it contains numbers and or .

I think that there is an issue with my stuff by I couldn't find it. In earlier version of my script it worked fine but in the latest version that I combined 2 different files with in total over 20 sheets, I have had this issue.

**Comment by jpecora716** [comment]
I run into this when trying to save the spreadsheet on top of a spreadsheet that is already open. The error can be very confusing.

**Comment by namnamir** [comment]
> 
> 
> I run into this when trying to save the spreadsheet on top of a spreadsheet that is already open. The error can be very confusing.

Please take a look at [this solution](https://github.com/dfinke/ImportExcel/issues/845). It may help your issue too.

**Comment by Jarrettgohh** [comment]
> 

> I run into this when trying to save the spreadsheet on top of a spreadsheet that is already open. The error can be very confusing.

Well yeah dumb mistake, but closing the file solves this issue. It would be great if there is a clearer error message. Thanks anyway!

**Comment by ChrisLegend95** [comment]
I found the same problem with Export-Excel

And I found out when I was using PS ISE I finger out why it did not work to save as I did before. I try to change it back to Export-CSV and it told me I did not have access to save to my C:\. 

After it, I saved my script and closed the PS ISE, and ran it again but as an Admin... and now it works fine to Export-Excel.

I hope this will help you all.

//Chris
